src/pages/workspaces/workspace/Data.js

```diff
@@ -666,6 +683,16 @@ const WorkspaceData = _.flow(
     setEntityMetadata(updatedMetadata)
   }
 
+  const searchAcrossTables = async (types, activeCrossTableTextFilter) => {
```

Comment:
This function could probably use some error handling.

Reply:
Added some error reporting/handling in ff3e71d9b1975670734b359d3049afe7d034ce30